cms/jinja2/templates/pages/analysis_page.html

```diff
@@ -0,0 +1,150 @@
+                            "itemsList": table_of_contents
+                        })
+                    }}
+                    {# fmt:on #}
+                {% endwith %}
+            </div>
+
+            <div class="ons-grid__col ons-col-8@m">
+                {# if there are no contact details on the page, we don't want the last
+                streamfield block to have a bottom margin. last_flush is used in stream_block.html #}
+                {% if page.contact_details %}
+                    {% set last_flush = False %}
+                {% else %}
+                    {% set last_flush = True %}
+                {% endif %}
+
+                {% include_block page.content %}
+
+                {% if page.show_cite_this_page %}
+                    <section id="cite-this-page">
+                        <h2 class="heading-2">{{ _("Cite this analysis") }}</h2>
+                        <p>
+                            {%- set cite_link -%}
+                                <a href="{{ fullpageurl(page) }}">{{ page.display_title }}</a>
+                            {% endset %}
+                            {% trans trimmed release_date=page.release_date|date("DATE_FORMAT"), cite_link=cite_link %}
+                                Office for National Statistics (ONS), released {{ release_date }}, ONS website, analysis, {{ cite_link }}
+                            {% endtrans %}
+                        </p>
+                    </section>
+                {% endif %}
+
+                {% if page.contact_details %}
+                    <section id="contact-details">
+                        <h2 class="heading-2">{{ _("Contact details") }}</h2>
+
+                        {% call onsTextIndent() %}
+                            <p class="contact-details__line">
+                                <strong>Name</strong>: {{ page.contact_details.name }}
+                            </p>
+                            <p class="contact-details__line">
+                                <strong>Email</strong>: <a href="mailto:{{ page.contact_details.email }}">{{ page.contact_details.email }}</a>
+                            </p>
+                            {% if page.contact_details.phone %}
+                                <p class="contact-details__line">
+                                    <strong>Telephone</strong>: <a href="tel:{{ page.contact_details.phone | replace(' ','')}}">{{ page.contact_details.phone }}</a>
+                                </p>
+                            {% endif %}
+                        {% endcall %}
+                    </section>
+                {% endif %}
+            </div>
+        </div>
+    </div>
+{% endblock %}
+
+{% block scripts %}
+    {{ super() }}
+    {% if page.has_equations %}
+        <script async src="{{ mathjax() }}"></script>
+    {% endif %}
+    {% if page.has_ons_embed %}
+        <script src="{{ static('js/third-party/pym.min.js') }}"></script>
+    {% endif %}
+{% endblock %}
```

Comment:
Make this a reusable partial?. As we will need this on other pages.

Reply:
49f9383